reports/test.html

```diff
@@ -0,0 +1,133 @@
+
+<!DOCTYPE html>
```

Comment:
should this file be committed?

Reply:
hmm idk it was commited on the old one - https://github.com/TBD54566975/web5-spec/blob/main/reports/test.html

Reply:
Weird. Let's drop it.